source/includes/_tipser-elements-analytics.md

```diff
@@ -76,7 +76,13 @@ For the instructions how to setup analytics.js script on your site and connect i
 
 The above example is just a very trivial example of integrating Tipser events with GA. For a more thorough implementation involving Enhanced Ecommerce extension, please refer to [this gist](https://gist.github.com/piotr-sobczyk/c652fd03488e4bd4003d66c763b5127d).
 
-To make it work, please make sure that Enhanced Ecommerce extension is enabled in you GA settings:
+To make it work, please make sure that you add `e-commerce plugin` from Google Analytics, by adding this line at the end of the snippet copied from your GA account: 
```

Comment:
To co tutaj piszesz jest specyficzne dla konkretnego sposobu implementacji ga na stronie POSa: analytics.js . Jeżeli ktoś używa np. gtag, to ten snippet mu nie pomoże.

Wg mnie warto wtrącić jeszcze zdanie typu: "For example, if you're using analytics.js script to integrate with Google Analytics, you need to add the line below (as described [here](https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#loadit)")